samples/Hystrix.Dotnet.Samples.AspNetCore/Controllers/HelloController.cs

```diff
@@ -19,10 +19,19 @@ public HelloController(IHystrixCommandFactory hystrixCommandFactory)
         
         [HttpGet]
         public async Task<IActionResult> Get()
-        { 
+        {
+            var firstCall = true;
+
             var result = await hystrixCommand.ExecuteAsync(
                 async () =>
                 {
+                    if(firstCall)
+                    {
+                        // On the first call we throw an exception to be able to illustrate Retry.
+                        firstCall = false;
+                        throw new InvalidOperationException();
```

Comment:
The blog post I read about retries in the original Hystrix library mentioned they exclusively did this in case of a network error, not for other types of errors. That makes sense, because in that case it's likely a retry will help, but for other types of errors either your request or the called system is at fault, which is likely to be corrected with an immediate retry.